_maps/configs/syndicate_gorlex_revenant.json

```diff
@@ -0,0 +1,46 @@
+{
+	"faction": "/datum/faction/syndicate",
+	"prefix": "ISV",
+	"namelists": [
+		"GORLEX",
+		"NATURAL_AGGRESSIVE",
+		"BEASTS",
+		"WEAPONS"
+	],
+	"map_name": "Revenant Placeholder",
+	"map_short_name": "Revenant-class",
+	"description": "Not finished.",
+	"tags": [
+		"RP Focus",
+		"Combat",
+		"Engineering"
+	],
+	"map_path": "_maps/shuttles/syndicate/syndicate_gorlex_revenant.dmm",
```

Comment:
Hey, rename it to `syndicate_hardliner_revenant.dmm`. The "gorlex" part is outdated, and is being slowly removed on the NGR side of things.